src/controllers/querriesController.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from "express";
+import { sendResponse } from "../utils/http.exception";
+import { insert_function, read_function } from "../utils/db_methods";
+import { querriesModelAttributes } from "../types/model";
+import { Querries } from "../database/models/querry";
+import { category_utils } from "../utils/controller";
+
+let queryId: string;
+export const createQuerry = async (req: Request, res: Response) => {
+	try {
+		const { firstName, lastName, email, subject, message } = req.body;
+
+		const newQuerry = await insert_function<Querries>("Querries", "create", {
+			firstName,
+			lastName,
+			email,
+			subject,
+			message,
+		});
+
+		return sendResponse(
+			res,
+			201,
+			"CREATED",
+			"Querry created successfully",
```

Comment:
You can show user a convenient message such as "Message sent" instead of  "Querry created successfully"

Reply:
cool